mongodb/activity/queryDocument/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/project-flogo/datastore-contrib/mongodb/activity/queryDocument
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/project-flogo/core v0.9.3
+	github.com/project-flogo/datastore-contrib/mongodb/connection v1.0.0
```

Comment:
I'm guessing this is needed for the test, if so, the version should be v0.1.0

Reply:
will use v0.1.0

Reply:
Done..